src/main/resources/view/HelpWindow.fxml

```diff
@@ -1,18 +1,41 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import javafx.geometry.Insets?>
 <?import javafx.scene.Scene?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
 <?import javafx.scene.image.Image?>
-<?import javafx.scene.web.WebView?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.stage.Stage?>
```

Comment:
Strange that `javafx.stage.Stage` is being imported here even though it is not used in the FXML file.

If it was generated by Gluon Scenebuilder then nevermind.

Reply:
Yeah it's generated by scenebuilder so I didn't touch it.